src/PostgREST/Error.hs

```diff
@@ -73,6 +74,10 @@ binaryFieldError =
   simpleError HT.status406 (toS (toMime CTOctetStream) <>
   " requested but a single column was not selected")
 
+connectionLostError :: Response
```

Comment:
Using the either type we could also remove this function and use only `apiRequestError`